.husky/pre-commit

```diff
@@ -1,4 +1,4 @@
 #!/bin/sh
 . "$(dirname "$0")/_/husky.sh"
 
-npm run husky_msg && npm run compile_css
+npm run husky_msg && npx lint-staged && npm run compile_css
```

Comment:
`lint-staged` was running at the wrong time -- it needs to run prior to `git commit`